### PR TITLE
fix link

### PR DESCRIPTION
**Pull request by vasu-1**
<hr>

## Description 📜

I have fixed the link

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [x] Other (If you choose other, Please mention changes below) 
    - [x] Fix-Document
<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [x] C/CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

Do check it out @Suvraneel

### CODE REVIEW

File: C_CPP/OOPs in C++/Encapsulation/readme.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 
-[Encapsualtion_in_c++](https://github.com/balasai45/winter-of-contributing/blob/e218ef26f277da8bdeda6cd6a86710f0a9ef2298/C_CPP/OOPs%20in%20C++/Encapsulation/Encapsulation.md)
+[Encapsualtion_in_c++](Encapsulation.md)
```
Comment (Suvraneel) [inline C_CPP/OOPs in C++/Encapsulation/readme.md:2]:
```suggestion
[Encapsulation_in_c++](Encapsulation.md)
```

**Review by Suvraneel — CHANGES_REQUESTED** [review]

**Comment by Suvraneel** [comment]
@vasu-1  Main direct kar deta hun

**Comment by vasu-1** [comment]
image

**Comment by vasu-1** [comment]
Koi baat ni !

jug jug jiyo !

**Comment by Suvraneel** [comment]
@vasu-1  
image

**Comment by vasu-1** [comment]
image

**Comment by vasu-1** [comment]
Bade Log

**Pull request closed without merging**